packages/orchestration/src/contracts/stakingAccountHolder.js

```diff
@@ -4,14 +4,18 @@ import {
   MsgDelegate,
   MsgDelegateResponse,
 } from '@agoric/cosmic-proto/cosmos/staking/v1beta1/tx.js';
+import {
```

Comment:
`src/contracts` suggests this contract is part of the product API. Is that the case? or is it an example?
It's a pattern we have come to regret in the zoe package: #7392

Maybe they belong in `tools/`?

Reply:
Now in `/src/examples` after rebasing